node/impl/full/eth.go

```diff
@@ -40,6 +40,7 @@ import (
 )
 
 var ErrUnsupported = errors.New("unsupported method")
+var RewardPercentileArrayLimit int = 100
```

Comment:
nit `maxEthFeeHistoryRewardPercentiles` is a little more descriptive (not a limit on the array size, a limit on something we're doing).

Also, `const ... = 100`, no need for this to be variable.

Reply:
Made appropriate changes.